src/routes/index.ts

```diff
@@ -1,10 +1,12 @@
-import { Router } from "express";
-import sampleRouter from "./sample";
+import { Router } from "express"
+import sampleRouter from "./sample"
+import authRouter from "./auth"
 
 // Create a new Router instance
-const router = Router();
+const router = Router()
 
 // Mount the routers
-router.use("/sample", sampleRouter);
+router.use("/sample", sampleRouter)
+router.use("/api/v1", authRouter)
```

Comment:
maybe we want to move the auth router to `/api/v1/auth`, we could also refactor and create a main api router for the `/api/v1` path